load-jetpack.php

```diff
@@ -54,6 +54,7 @@ function jetpack_should_use_minified_assets() {
 require_once JETPACK__PLUGIN_DIR . 'functions.global.php';
 require_once JETPACK__PLUGIN_DIR . 'functions.compat.php';
 require_once JETPACK__PLUGIN_DIR . 'functions.gallery.php';
+require_once JETPACK__PLUGIN_DIR . 'functions.cookies.php';
```

Comment:
Since the file is meant to be used only when a site is connected to WordPress.com already, do you think we could load it here instead?
https://github.com/Automattic/jetpack/blob/2b738920b21af3dca296e46422feab45406edaeb/modules/module-extras.php#L48

Reply:
@kwight tried something similar in #14356 but it didn't work because those files are loaded after `pluggable.php`: https://github.com/Automattic/jetpack/pull/14356#issuecomment-574892429

Reply:
btw, should we rename it something else to make it clear it will only affect WP.com-connected sites? i.e. `functions.wpcom-cookies.php`.

Reply:
on second thought, the file provides a `jetpack_shim_setcookie` function that can be used anywhere in Jetpack, so it might be better to stick with the more generic `function.cookies.php` name.